kubernetes/yaml/vsphere-windows/kustomization.yaml

```diff
@@ -10,3 +10,16 @@ patches:
       group: apps
       version: v1
       name: kibishii-deployment
+  - target:
+      kind: Pod
+      version: v1
+      name: jump-pad
+    patch: |-
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        name: jump-pad
+      spec:
+        containers:
+          - name: jump-pad
+            image: vsphere-docker-dev-local.usw5.packages.broadcom.com/vkd/velero/jump-pad:main-362c5ab-07.05.2025.11.09.14
```

Comment:
1. Make this configurable to avoid the internal URLs
2. We can use the images from Dockerhub via our internal proxy, so no need to push the images to the internal registry again

Reply:
Modified.